NuRadioReco/framework/parameters.py

```diff
@@ -63,6 +63,26 @@ class channelParameters(Enum):
     Vrms_NuRadioMC_simulation = 19  #: the noise rms used in the MC simulation
     bandwidth_NuRadioMC_simulation = 20  #: the integrated channel response (=bandwidth for signal chains without amplification) used in the MC simulation
     Vrms_trigger_NuRadioMC_simulation = 21  #: the noise rms of the trigger channels (optional) used in the MC simulation
+    root_power_ratio = 22 #: the root power ratio (float) 
+
+class channelParametersRNOG(Enum):
+    # RNO-G specific channel parameters
+
+    # FS: I did not start with a negative parameter on the 1, hence I chose 100
+    glitch = 100 #: True if channel is likely to have a glitch. See 'NuRadioReco.modules.RNO_G.channelGlitchDetector'
+    glitch_test_statistic = 101 #: Numerical value delivered by the glitch detector. Positive values indicate a likely glitch.
+
+class eventParametersRNOG(Enum):
+    max_corr_coords = 4 #: azimuth, zenith corresponding to max correlation
+    max_corr = 5 #: maximum correlation value obtained through reconstruction
+    csw_snr = 6 #: signal to noise ratio of coherently summed waveform
+    csw_rpr = 7 #: root power ratio of coherently summed waveform
+    csw_hilbert_snr = 8 #: signal to noise ratio of hilberted coherently summed waveform
+    csw_impulsivity = 9 #: coherently summed waveform's impulsivity and other statistical measures (dict)
+    surf_corr_ratio = 10 #: ratio comparing max surface correlation to max overall correlation
+    max_surf_corr = 11 #: maximum surface correlation
```

Comment:
Important parameters! I was just wondering if we should move them to `stationParametersRNOG` and maybe rename them to have less acronyms?